### DISCUSSION TITLE
Editing MTEX plots

### DISCUSSION THREAD

**Discussion by huseyinhzl** [mtex-toolbox/mtex#1692] [Ask Anything] [answered]
Hi all,

I am us
ng MTEX 5.8.2 and ORTools 2.0.2 for parent grain reconstruction and I have two problems that I have not been able to solve by myself. 

1. How can I remove the bounding box around the graphs that I have produced with MTEX? I have tried the box off command when I am plotting but it did not help, so I am wondering if there is a solution for this.
2. How can I change the line color in plotMap_variants command? Currently, it is black but I want to make it white so that my grains will be a better look. I put an example image of variant plot

![1_21](https://user-images.githubusercontent.com/120036622/208461300-2de9da65-dba8-4194-8b4a-508830a4f790.jpeg)

Thank you

**Comment by frankNiessen** [comment, marked as answer]
Hi Huseyin, 

I have answered your request in the [ORTools](https://github.com/ORTools4MTEX/ORTools/discussions/27) discussion board.

Best wishes
Frank
